packages/server/src/modules/messages/infra/http/routes/message.ts

```diff
@@ -116,6 +114,27 @@ messageRouter.post('/:id/send', async (req, res) => {
   return res.json(message);
 });
 
+messageRouter.post('/:id/send-to-group', async (req, res) => {
```

Comment:
We shouldn't have a route to send a message to the groups, we have to use the same route.